theme/colors/light.css

```diff
@@ -103,9 +103,9 @@
 	--gnome-trough-active-background: color-mix(in srgb, currentColor 25%, transparent);
 
 	/* Buttons */
-	--gnome-button-background: color-mix(in srgb, currentColor 10%, transparent);
-	--gnome-button-hover-background: color-mix(in srgb, currentColor 15%, transparent);
-	--gnome-button-active-background: color-mix(in srgb, currentColor 30%, transparent);
+	--gnome-button-background: #ebebeb;
+	--gnome-button-hover-background: #e0e0e0;
+	--gnome-button-active-background: #dedede;
```

Comment:
We don't want opaque colors here, otherwise when we recolor the headebar in private browsing it will look bad.

Currently colors are base on the libadwaita 1.3 ones. If colors have changed we must check the libadwaita 1.4 sass files and update the opacity o whatever has changed.

Reply:
Actually, this is not used by flat buttons (they use `--gnome-hover-color`), but the point is the same, we want to use translucent colors when libadwaita uses them.

Reply:
Thanks! Good to know. I'll try to see if I can find time to update it further this weekend.

Reply:
I reverted those specific color changes. After the other color changes, those wind up being the correct colors anyway. (I checked with a screenshot afterward and compared with Epiphany.)